.github/workflows/test_coverage.yml

```diff
@@ -0,0 +1,24 @@
+name: Test Coverage
+on:
+  push:
+    branches:
+      - develop
+      - main
```

Comment:
Both other yml files callout develop and main, so I figured it'd be good for uniformity to do the same within this file as well.

Reply:
I reverted this change because we only track coverage on the develop branch.